pkg/rancher-desktop/backend/wsl.ts

```diff
@@ -713,14 +713,19 @@ export default class WSLBackend extends events.EventEmitter implements VMBackend
     const credsPath = getServerCredentialsPath();
 
     try {
-      const vtunnelPeerServer = '127.0.0.1:3030';
+      const vtunnelPeerServerAddr = '127.0.0.1:3030';
+      const credentialServerAddr = '192.168.127.254:6109';
+      // When networkTunnel is enabled we talk directly to the host which is assigned
+      // with 192.168.127.254 static address. Otherwise, we talk to the vtunnel peer
```

Comment:
Not for this PR, perhaps, but it may be useful to have a `const HOST_IP = '192.168.127.254';` later and move (some of) the comment there.